content/en/changelogs/1.30.1-changelog.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Spinnaker Release 1.30.1
+date: 2023-04-07 03:46:41 +0000
+major_minor: 1.30
+version: 1.30.1
+---
+
+## [Clouddriver](#clouddriver) 5.80.1
```

Comment:
It's sort of hard to see here since it happened in kork.  Is it worth calling out something explicit about the SAML issue?